test/runtests.jl

```diff
@@ -0,0 +1,84 @@
+using SparseArrays
```

Comment:
Nice we have an example here! 
I tried run in several times, by playing with the different parameters. Apparently the two algorithms (dual decomposition and ADMM) are very slow to converge, even after 5,000 iterations. That surprises me, as the example implemented here is quite simple.